### PR TITLE
chore: remove beta tags from CLI and docs

### PR DESCRIPTION
**Pull request by pvorozhe**
🎉 Thanks for submitting a pull request! 🎉

#### Summary

Fixes [pillar-workflow/191](https://github.com/netlify/pillar-workflow/issues/919) (1/2)

---

For us to review and ship your PR efficiently, please perform the following steps:

- [ ] Open a [bug/issue](https://github.com/netlify/cli/issues/new/choose) before writing your code 🧑‍💻. This ensures we can discuss the changes and get feedback from everyone that should be involved. If you\`re fixing a typo or something that\`s on fire 🔥 (e.g. incident related), you can skip this step.
- [ ] Read the [contribution guidelines](../CONTRIBUTING.md) 📖. This ensures your code follows our style guide and
      passes our tests.
- [ ] Update or add tests (if any source code was changed or added) 🧪
- [ ] Update or add documentation (if features were changed or added) 📝
- [ ] Make sure the status checks below are successful ✅

**A picture of a cute animal (not mandatory, but encouraged)**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -344,6 +344,7 @@
     "husky": "^8.0.0",
     "ini": "^2.0.0",
     "mock-fs": "^5.1.2",
+    "netlify-cli": "^12.0.11",
```
Comment (danez) [inline package.json:347]: I don't think we should have the CLI as dependency in the CLI :D

**Review by danez — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (lukasholzer) [reply package.json:347]: oof :D

Comment (pvorozhe) [reply package.json:347]: Good catch :my-bad:

**Review by akahn — DISMISSED** [review]

**Review by codebyuma — COMMENTED** [review]
The copy update looks good to me 🎉

**Review by danez — CHANGES_REQUESTED** [review]
The file `npm-shrinkwrap.json` still has changes, could you revert those?

**Comment by pvorozhe** [comment]
> The file `npm-shrinkwrap.json` still has changes, could you revert those?
I reverted the changes to main branch, they still show up in the diff. not sure why

**Review by danez — APPROVED** [review]

**Comment by Skn0tt** [comment]
this PR is failing on main, due to a snapshot not being updated: https://github.com/netlify/cli/actions/runs/3378626827/jobs/5609083160#step:11:136. I wonder why the CI here didn't catch it 🤔 

I'll fix this as part of https://github.com/netlify/cli/pull/5200, but we should ensure that our CI catches these kinds of things in the future.

**Comment by Skn0tt** [comment]
I have a hunch that  the `chore: update contributors field` commits are resulting in green runs, because the `test:affected`  strategy is resulting in untouched files not being tested. Maybe we could adapt the `test:affected` strategy to not only test files touched by recent commits, but files touched in `HEAD..PR HEAD`

**Comment by Skn0tt** [comment]
@lukasholzer maybe you have an idea

**Pull request merged**